### ISSUE TITLE
Transfer learning for Masknet?

### ISSUE THREAD

**Issue by kidpaul94** [vinits5/learning3d#10]
Hello again!

I was trying to use Masknet to do inlier estimation for my own object. But the model didn't show good performance in the estimation even if I normalized both point clouds and they were already manually registered. This might have happened because my source miss almost 60% of geometrical feature that my target has (due to sensor noise and occlusion) or point density between pcs are different. But, I also presume that this can happen because my own object is different from objects used for training Masknet. Could you provide me some information to do transfer learning for Masknet (if possible)?

**Comment by kidpaul94** [comment]
Never mind, I found your another repo (masknet) to do this.

**Comment by omar-ihab-ali** [comment]
Hi, I am also trying to use Masknet and it didn't show good performance on my usecase.

@kidpaul94 Did the fine-tuning made a difference in the results. Generating a training dataset for my usecase is a bit complex, so wanted to get some insights before starting. Thanks in advance :)
